pyquest/drawer.py

```diff
@@ -143,15 +267,13 @@ def get_operated_qubits(gate, num_qubits):
 def get_num_qubits(gates):
 
     # find the biggest indexed target/control qubit among explicitly-targeted gates
-    return 1 + max(
```

Comment:
Some stylistic changes are inconsistent. Here you opt to compact the generator from three lines to one...